custom_components/ramses_cc/__init__.py

```diff
@@ -207,38 +206,16 @@ def extra_state_attributes(self) -> dict[str, Any]:
     async def async_added_to_hass(self) -> None:
         """Run when entity about to be added to hass."""
         self._broker._entities[self.unique_id] = self
-        async_dispatcher_connect(self.hass, DOMAIN, self.async_handle_dispatch)
```

Comment:
Is it right to remove this?

Reply:
It's replaced with the more explicit `SIGNAL_UPDATE`.

Reply:
So, this is the first update after adding the entity.